test/WebJobs.Extensions.Tests/Extensions/Http/HttpTriggerBindingTests.cs

```diff
@@ -29,9 +29,12 @@ public class HttpTriggerBindingTests
             request.Content = new StringContent(input);
             var bindingData = await HttpTriggerAttributeBindingProvider.HttpTriggerBinding.GetRequestBindingDataAsync(request);
 
-            Assert.Equal(4, bindingData.Count);
+            Assert.Equal(5, bindingData.Count);
             Assert.Equal("testing", bindingData["test"]);
             Assert.Equal("123", bindingData["baz"]);
+
+            dynamic nestedObject = bindingData["nestedObject"];
```

Comment:
>dynamic [](start = 12, length = 7)

Can we make this a static type instead of dynamic.  Is it JObject? Dict? Expando? 
This is a test, so we can enforce the strict behavior and catch it if it changes.  Dynamic is too loose.